datajack-api/src/test/java/ru/sbtqa/tag/datajack/callback/SampleDataGensCallback.java

```diff
@@ -30,6 +30,7 @@ public Object call(CallbackData callbackData) {
             String genResult = null;
 
             try {
+                Thread.sleep(1);
```

Comment:
dafuq?

Reply:
WAT?! It`s for different generation results (timestamp increment).